robosuite/demos/demo_control.py

```diff
@@ -89,10 +89,22 @@
     joint_dim = 6 if options["robots"] == "UR5e" else (16 if options["robots"] == "GR1" else 7)
 
     # Choose controller
-    controller_name = choose_controller()
+    controller_name = choose_part_controller()
+    part_controller_config = suite.load_part_controller_config(default_controller=controller_name)
```

Comment:
Could we prompt the user for the actual body part? E.g. humanoid might have multiple body parts and I suspect this script may fail for e.g. GR1FixedLowerBody